tests/README.md

```diff
@@ -44,6 +44,9 @@ export TEST_BASE_DIR=WORKING_DIR
 export TEST_SCRATCH_DEVICE=/dev/vdb
 ```
 
+Change the `TEST_SCRATCH_DEVICE` and point it to the correct block device name based on the driver that's being used.
+For example `/dev/sdb` in case the driver is `virtualbox`.
+
 Please note that the integration tests erase the contents of TEST_SCRATCH_DEVICE.
```

Comment:
While you are at it, please add backticks to the `TEST_SCRATCH_DEVICE` and on line 55/58 below the path `tests/integration/_output`.
```suggestion
Please note that the integration tests erase the contents of `TEST_SCRATCH_DEVICE`.
```